multiplayer/src/services/gameClient.ts

```diff
@@ -22,12 +23,13 @@ import {
     playerJoinedAsync,
     playerLeftAsync,
     setGameMetadataAsync,
+    gameOverAsync,
 } from "../epics";
 
 const GAME_HOST_PROD = "https://mp.makecode.com";
 const GAME_HOST_STAGING = "https://multiplayer.staging.pxt.io";
 const GAME_HOST_LOCALHOST = "http://localhost:8082";
-const GAME_HOST_DEV = GAME_HOST_STAGING;
+const GAME_HOST_DEV = GAME_HOST_LOCALHOST;
```

Comment:
Itd be nice if there was a gitignorenextline

Reply:
Indeed. Or, maybe we can control this with an environment variable instead.